atomsci/ddm/pipeline/compare_models.py

```diff
@@ -1641,7 +1638,7 @@ def get_dataset_models(collection_names, filter_dict={}):
                 "match_metadata": query_filter
             }
 
-            print('Querying models in collection %s for dataset %s, %s' % (collection_name, bucket, dset_key))
+            print('Querying models in collection %s for dataset %s, %s' % (collection_name, dset_dict['bucket'], dset_dict['dataset_key']))
```

Comment:
this would have thrown an error so this function has never been used or tested. Seems useful if